common/flatpak-dir.c

```diff
@@ -7808,12 +8058,38 @@ flatpak_dir_install (FlatpakDir          *self,
                                  flatpak_flags,
                                  OSTREE_REPO_PULL_FLAGS_MIRROR,
                                  progress, cancellable, error))
-            return FALSE;
+            {
+              if (revokefs_fuse)
```

Comment:
It's sad to copy-paste this code here and below.

Reply:
Yeah. maybe this is intentional as `flatpak_dir_pull` definition is quite long. I am not sure if it is kept for readability rather than having:
```
if (!flatpak_dir_pull (self, state, ref, opt_commit, NULL, subpaths,
                       child_repo,
                       flatpak_flags,
                       OSTREE_REPO_PULL_FLAGS_MIRROR,
                       progress, cancellable, error) &&
    child_repo_ensure_summary (child_repo, state, cancellable, error))
{
...
}
```
There are a couple of places in `common/flatpak-dir.c` with  "`flatpak_dir_pull` + `child_repo_ensure_summary`" combo. Maybe we should look at them all-together in a separate trivial change if possible.